src/com/esotericsoftware/kryo/serializers/TaggedFieldSerializerConfig.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2008, Nathan Sweet
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following
+ * conditions are met:
+ *
+ * - Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+ * - Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided with the distribution.
+ * - Neither the name of Esoteric Software nor the names of its contributors may be used to endorse or promote products derived
+ * from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+ * BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. */
+
+package com.esotericsoftware.kryo.serializers;
+
+import static com.esotericsoftware.minlog.Log.TRACE;
+import static com.esotericsoftware.minlog.Log.trace;
+
+/** Configuration for TaggedFieldSerializer instances. */
+public class TaggedFieldSerializerConfig extends FieldSerializerConfig {
+
+    /** ignore unknown field tags when using TaggedFieldSerializer. */
+    private boolean ignoreUnknownTags = false;
+
+    /** Tells Kryo, if should ignore unknown field tags when using TaggedFieldSerializer. Already existing serializer instances
+     * are not affected by this setting.
+     *
+     * <p>
+     * By default, Kryo will throw KryoException if encounters unknown field tags.
+     * </p>
+     *
+     * @param ignoreUnknownTags if true, unknown field tags will be ignored. Otherwise KryoException will be thrown */
+    public void setIgnoreUnknownTags (boolean ignoreUnknownTags) {
+        this.ignoreUnknownTags = ignoreUnknownTags;
+        if (TRACE) trace("kryo.FieldSerializerConfig", "setIgnoreUnknownTags: " + ignoreUnknownTags);
```

Comment:
Instead of "kryo.FieldSerializerConfig" it should be "kryo.TaggedFieldSerializerConfig".